Source/JavaScriptCore/bytecode/UnlinkedMetadataTable.cpp

```diff
@@ -32,9 +32,11 @@ namespace JSC {
 
 DEFINE_ALLOCATOR_WITH_HEAP_IDENTIFIER(MetadataTable);
 
-#define JSC_ALIGNMENT_CHECK(size) static_assert(size <= UnlinkedMetadataTable::s_maxMetadataAlignment);
+static_assert((UnlinkedMetadataTable::s_maxMetadataAlignment >=
+#define JSC_ALIGNMENT_CHECK(size) size) && (size >=
 FOR_EACH_BYTECODE_METADATA_ALIGNMENT(JSC_ALIGNMENT_CHECK)
 #undef JSC_ALIGNMENT_CHECK
+1));
```

Comment:
Looks like GCC is not happy with this

Reply:
Yeah, it's probably because when computing the order I assumed that pointers were 8-byte aligned. I might just skip both new assertions on 32-bit since we will still align if necessary.